crates/bevy_core_pipeline/src/contrast_adaptive_sharpening/mod.rs

```diff
@@ -0,0 +1,265 @@
+use crate::{core_2d, core_3d, fullscreen_vertex_shader::fullscreen_shader_vertex_state};
+use bevy_app::prelude::*;
+use bevy_asset::{load_internal_asset, HandleUntyped};
+use bevy_ecs::{prelude::*, query::QueryItem};
+use bevy_reflect::{Reflect, TypeUuid};
+use bevy_render::{
+    extract_component::{ExtractComponent, ExtractComponentPlugin, UniformComponentPlugin},
+    prelude::Camera,
+    render_graph::RenderGraph,
+    render_resource::*,
+    renderer::RenderDevice,
+    texture::BevyDefault,
+    view::{ExtractedView, ViewTarget},
+    RenderApp, RenderStage,
+};
+
+mod node;
+
+pub use node::ContrastAdaptiveSharpeningNode;
+
+/// Applies a contrast adaptive sharpening filter to the camera.
```

Comment:
Needs a short paragraph describing the effect. Something about how it sharpens an image, which is useful in conjunction with effects like FXAA or TAA that soften things.